assets/xml/scenes/overworld/spot00.xml

```diff
@@ -1,6 +1,8 @@
 <Root>
     <File Name="spot00_scene" Segment="2">
-        <Scene Name="spot00_scene"/>
+        <Scene Name="spot00_scene">
+            <CutsceneHint Offset="0xF870"/>
+        </Scene>
```

Comment:
Please don't use the Hint system. It is deprecated and is going to be removed.

Could you also give a proper name to this Cs?
```suggestion
        <Cutscene Name=gSpot00Cs_00F870" Offset="0xF870"/>
        <Scene Name="spot00_scene" Offset="0x0">
        </Scene>
```